KidsIdKit.Shared/Pages/Index.razor

```diff
@@ -1,12 +1,15 @@
 ﻿@page "/"
```

Comment:
Why are there changes to this file?

Reply:
Old code when we were fiddling around with pdf libraries. Reverted.